public/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Welcome to Vue</title>
+  <title>Welcome to Face Overlay</title>
   <script src="https://unpkg.com/vue"></script>
   <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 </head>
 <body>
   <div id="app">
-
     <div class="container" style="margin-top: 3%; margin-left: 2%;">
+      <h1>Welcome to Face Overlay App</h1>
       <div class="col-md-6">
         <div>
           <div class="col-md-6">
-          <img src="http://res.cloudinary.com/unicodeveloper/image/upload/v1505797868/glasses.png" width="200" height="100" />
+          <img src="http://res.cloudinary.com/og-tech/image/upload/v1507385206/glasses.png" width="200" height="100" />
           <span> Glasses </span>
           </div>
           <div class="col-md-6">
-          <img src="http://res.cloudinary.com/unicodeveloper/image/upload/v1505794374/oie_transparent.png" width="200" height="100" />
+          <img src="http://res.cloudinary.com/og-tech/image/upload/v1507379406/harlequinmask.jpg" width="200" height="100" />
```

Comment:
Same as here. Why did you replace the image? This is not really necessary. 

Any reason why you changed it?

Reply:
the link to the image was broken here, that's why this was changed

Reply:
Okay. Make sense.

Reply:
I found the link that works `http://res.cloudinary.com/unicodeveloper/image/upload/v1505794374/harlequinmask.png` So I'll change it to that too